webapp/channels/src/components/size_aware_image.tsx

```diff
@@ -248,15 +271,9 @@ export default class SizeAwareImage extends React.PureComponent {
                     onClick={this.copyLinkToAsset}
                 >
                     {this.state.linkCopiedRecently ? (
-                        <CheckIcon
-                            className={'svg-check style--none'}
```

Comment:
so why was the classname removed?

Reply:
There is no property of className on the icons

Reply:
i see but is the classname propagated to the DOM, can you run the app and verify that?

Reply:
YEah ! cool i will do that

Reply:
![image](https://github.com/mattermost/mattermost/assets/93595710/4f9e3523-66a0-4312-b4cc-a2fc19f6e705)
There is no declaration of className prop for these icons ! just checked

Reply:
declaration can be different than actual implementation, but if you see the source code of the component it seems that the rest of the properties are being spread on to the dom element, i even cross checked it within the app

![image](https://github.com/mattermost/mattermost/assets/17708702/e3c7b70f-0e57-4398-98df-e8cd218bc463)

Reply:
shiuld i revert it ?

Reply:
but it will give type error ! before it was js ! so it was not a problem ! what to do?

Reply:
yes please

Reply:
what about the type error?

Reply:
Are you open to adding the class name type in compass-icon?

Reply:
yeah ! i will create an issue in that repo! and make a pr !

Reply:
Yeah I will do that ! Just checked the repo and got the solution ! 
I will create a issue first and make a pr ! is that fine?

Reply:
yup that works

Reply:
@M-ZubairAhmed I have put cc in the pr of compass-icon !